homeassistant/components/plugwise/const.py

```diff
@@ -30,16 +30,7 @@
     Platform.SENSOR,
     Platform.SWITCH,
 ]
-ZEROCONF_MAP: Final[dict[str, str]] = {
-    "smile": "P1",
-    "smile_thermo": "Anna",
-    "smile_open_therm": "Adam",
-    "stretch": "Stretch",
-}
```

Comment:
I understand these changes (sorting 🔠 )

But they make PRs harder to follow and unneeded large. Please do refactoring in a separate PR to keep PRs to a minimum.